cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -1559,7 +1561,7 @@ instance HasTypeProxy Certificate where
     proxyToAsType _ = AsCertificate
 
 instance HasTextEnvelope Certificate where
-    textEnvelopeType _ = "Certificate"
+    textEnvelopeType _ = "CertificateShelley"
```

Comment:
Should I suffix with `*Shelley` even though there's only one kind of certificate right now?